crates/ethcore/sync/src/chain/mod.rs

```diff
@@ -156,6 +156,8 @@ impl From<DecoderError> for PacketProcessError {
     }
 }
 
+/// Version 65 of the Ethereum protocol and number of packet IDs reserved by the protocol (packet count).
+pub const ETH_PROTOCOL_VERSION_65: (u8, u8) = (65, 0x11);
```

Comment:
The second number is the count of messages that we have for that protocol. Should be greater then eth/64

Reply:
Are you sure? This was already questioned in the old PR and review, but I think 0x11 is correct, as it is the count of reserved packet ids, not of actually used ones. (Neither protocol actually has 17 messages, but both use the range of 0x00 to 0x10, so 0x11. Afaict the number is not used anywhere.)

Reply:
It is used here: https://github.com/openethereum/openethereum/blob/582bca385fedb1af682e989e5bcc6b3b2cf53028/crates/net/network-devp2p/src/session.rs#L458-L466
`packet_count` is that last number.

But you are right, I didn't see that message id's have a gap, and new packet id's are filled in that gap without increasing the last message id. Setting 0x11 is okay.